public/pages/DetectorConfig/containers/DetectorConfig.tsx

```diff
@@ -40,20 +40,22 @@ export const DetectorConfig = (props: DetectorConfigProps) => {
 
   return (
     <EuiPage style={{ marginTop: '16px', paddingTop: '0px' }}>
-      <EuiPageBody>
-        <EuiSpacer size="l" />
-        <MetaData
-          detectorId={props.detectorId}
-          detector={detector}
-          onEditDetector={props.onEditDetector}
-        />
-        <EuiSpacer />
-        <Features
-          detectorId={props.detectorId}
-          detector={detector}
-          onEditFeatures={props.onEditFeatures}
-        />
-      </EuiPageBody>
+      {detector ? (
+        <EuiPageBody>
+          <EuiSpacer size="l" />
+          <MetaData
+            detectorId={props.detectorId}
+            detector={detector}
+            onEditDetector={props.onEditDetector}
+          />
+          <EuiSpacer />
+          <Features
+            detectorId={props.detectorId}
+            detector={detector}
+            onEditFeatures={props.onEditFeatures}
+          />
+        </EuiPageBody>
+      ) : null}
```

Comment:
Do we want to display something here that would give more info to the user, such as if a detector's info is loading or if there was an error retrieving the detector?

Reply:
Good point, will add a loading state